generate/layout.go

```diff
@@ -790,16 +864,92 @@ var layout = apiInfo{
 	"RegistrationService": {
 		"registerOauthProvider",
 	},
-	"ShutdownService": {
-		"readyForShutdown",
-	},
```

Comment:
any specific reason to remove this? @phsm as the generate.go file has a reference to this api, causing issues:

```
test/ShutdownService_test.go:42:15: client.Shutdown undefined (type *cloudstack.CloudStackClient has no field or method Shutdown)
test/ShutdownService_test.go:43:20: client.Shutdown undefined (type *cloudstack.CloudStackClient has no field or method Shutdown)
```

Reply:
The reason was to make the layout structure to follow the structure from here: https://cloudstack.apache.org/api/apidocs-4.20/

The "readyForShutdown" API call looks to be a part of ManagementService now.